Library/Homebrew/sorbet/rbi/hidden-definitions/hidden.rbi

```diff
@@ -5874,15 +5677,6 @@ module Readline
   def self.completion_quote_character(); end
 end
 
-class ReporterHub
-  def empty?(*args, &block); end
```

Comment:
Since this method is no longer implicitly declared, i've removed the delegation and added an explicit def in a commit to this PR.

(I'm poking around for an explanation for why this behavior has changed, but no clue. Hardware change is an interesting theory, but i assume most of us have been using `brew typecheck --update` on newer hardware…)